consensus/checks.go

```diff
@@ -12,36 +13,36 @@ const MaxBlockNumDiff = 100
 
 func (consensus *Consensus) validatorSanityChecks(msg *msg_pb.Message) bool {
 	if msg.GetConsensus() == nil {
-		consensus.getLogger().Warn().Msg("[validatorSanityChecks] malformed message")
+		utils.Logger().Warn().Msg("[validatorSanityChecks] malformed message")
```

Comment:
let's not change these loggers for now. Consensus logger logs the internal state of consensus, it's useful to troubleshoot hard bugs.

Reply:
no, this logger was causing data race conditions, not adding value, caller should add what they need to plain logger

Reply:
So it crashed on the consensus logger? Ok, never seen it before though. How did you reveal so many race conditions now?

Reply:
by using our amazing golang tools haha